ecosystem-overview.md

```diff
@@ -0,0 +1,108 @@
+If you are an architect of a framework wishing to take advantage of speed 
+and multi-device support of nGraph Compiler, please refer to “how to 
+connect custom framework” section on this page. 
+
+
+|  Framework & Runtime       | Supported          |  Validated 
+|----------------------------|--------------------|-------------
+| TensorFlow* 1.12           | :heavy_check_mark: |  :heavy_check_mark:
+| MXNet* 1.4                 | :heavy_check_mark: |  :heavy_check_mark:
+| ONNX 1.3                   | :heavy_check_mark: |  :heavy_check_mark:
+| ONNX Runtime  Functional   | Functional         |  No
+| PyTorch (via ONNXIFI)      | Functional         |  No
+| PaddlePaddle               | Functional         |  No
+
+
+
+## Hardware & backend support
+
+The current release of nGraph primarily focuses on accelerating inference 
+performance on CPU. However we are also working on adding support for more 
+hardware and backends. As with the frameworks, we believe in providing 
+freedom to AI developers to deploy their deep learning workloads to the 
+desired hardware without a lock in. We currently have functioning backends 
+for Intel, Nvidia*, and AMD* GPU either leveraging kernel libraries 
+such as clDNN and cuDNN directly or utilizing PlaidML to compile for codegen 
+and emit OpenCL, OpenGL, LLVM, Cuda, and Metal. (Please refer to architecture 
+and features section to learn more about how we plan to take advantage of 
+both solutions using hybrid transformer). In the similar time frame, we plan 
+to release multinode support. 
+
+We are excited about providing support for our upcoming deep learning 
+accelerators such as NNP (Neural Network Processor) via nGraph compiler 
+stack, and early adopters will be able test them in 2019.
+
+
+
+| Backend                                       | supported         
+|-----------------------------------------------|-------------------
+| Intel® Architecture CPU                       | :heavy_check_mark:               
+| Intel® Architecture GPUs                      | Functional via clDNN and PlaidML      
+| AMD* GPUs                                     | Functional via PlaidML                 
+| Nvidia* GPUs                                  | Functional via cuDNN and PlaidML        
+| Intel® Nervana™ Neural Network Processor (NNP)| Functional               
+| [Upcoming DL accelerators]                    | see details on [Upcoming DL accelerators]       
+
+
+## How do I connect a framework?
+
+The nGraph Library manages framework bridges for some of the more widely-known 
+frameworks. A bridge acts as an intermediary between the nGraph core and the 
+framework, and the result is a function that can be compiled from a framework. 
+A fully-compiled function that makes use of bridge code thus becomes a "function 
+graph", or what we sometimes call an nGraph graph.
+
+**Note** Low-level nGraph APIs are not accessible dynamically via bridge code; 
```

Comment:
I don't know what this is trying to say, but it's currently wrong.

Reply:
okay, this whole paragraph is getting deleted.